libs/designsystem/src/lib/components/button/button.component.scss

```
@@ -1,8 +1,19 @@
 @import '../../scss/utils';
 
+$button-width: (
+  sm: 44px,
+  md: 88px,
+  lg: 220px,
+) !default;
+
+@function button-width($key: 'md') {
```

Don't see the need for a default...
```suggestion
@function button-width($key) {
```